staging/src/k8s.io/noderesourcetopology-api/manifests/crd-instance-example.yaml

```diff
@@ -0,0 +1,45 @@
+---
+apiVersion: topology.node.k8s.io/v1alpha1
+kind: NodeResourceTopology
```

Comment:
Is there a statement of objectives and requirements around this API?

Reply:
it's in KEP mentioned in message